tests/installers/test_download_installer.py

```
@@ -302,11 +302,79 @@ async def test_too_many_redirects(self, tmp_path):
         """Exceeding max redirects must raise ValueError."""
         dest = tmp_path / "out.bin"
         redirect = _FakeResponse(302, body=b"",
-                                  headers={"location": "https://safe.example/loop"})
+                                   headers={"location": "https://safe.example/loop"})
 
         mock_client, mock_send = _mock_async_client(redirect)
```

Ruff (0.16.2)

[warning] Unpacked variable `mock_send` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)